proto/doc.go

```diff
@@ -39,7 +39,73 @@
 //	                    |
 //	         ======= Dataplane ==========
 //
-// Overview
+//
+// Data-model Overview
+//
+// The data-model used on the dataplane driver API uses similar concepts to
+// the main "datastore" data-model (such as host/workload endpoints, policies
+// and profiles).  However, the calculation engine does some pre-calculation
+// that simplifies the job of the dataplane driver:
```

Comment:
And it uses different exact terminology in various places (e.g. untracked vs doNotTrack).  Is that worth mentioning as an overall point?